### PR TITLE
Hollow/new changes

### PR DESCRIPTION
**Pull request by abbclas**
The conflicts are resolved and ready to be merged to main

### CODE REVIEW

File: app/Scenes/NIVMIZZPlayer/Player.tscn [hunk 1]
```diff
@@ -0,0 +1,37 @@
+[gd_scene load_steps=6 format=2]
+
```
Comment (noh0st) [inline app/Scenes/NIVMIZZPlayer/Player.tscn]: we shouldn't have a folder called 'nivmizzplayer'. can you reconcile your changes with the existing player?

File: app/Main.gd [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	# initialize health UI with player
+	
+	
+	pass # Replace with function body.
+
+
+# Called every frame. 'delta' is the elapsed time since the previous frame.
+#func _process(delta):
```
Comment (noh0st) [inline app/Main.gd]: this file is outside of the scripts folder and also doesnt seem to do anything

File: app/Scripts/HealthUIController.gd [hunk 3]
```diff
@@ -4,15 +4,22 @@ extends CanvasLayer
...
-
+	staminaLaber.text = "Stamina: %s" % 100	
 # Called every frame. 'delta' is the elapsed time since the previous frame.
 #func _process(delta):
 #	pass
 
-func _on_Player_health_update(amount: int) -> void:
-	healthLabel.text = "Health: %d" % amount
+
+func _on_Player_health_update(new_value):
```
Comment (noh0st) [inline app/Scripts/HealthUIController.gd:18]: not going to be super strict about this cause it's just a game jam but i think typing hints can help catch bugs and make code more readable

File: app/project.godot [hunk 4]
```diff
@@ -13,13 +13,35 @@ config_version=4
 config/name="jump-ship"
 run/main_scene="res://Scenes/MainMenu.tscn"
 config/icon="res://icon.png"
+<<<<<<<Updatedupstreamrun/main_scene="res://Scenes/MainMenu.tscn"
+<<<<<<<HEADrun/main_scene="res://Scenes/Main.tscn"
```
Comment (noh0st) [inline app/project.godot]: shouldnt be adding this

**Comment by noh0st** [comment]
can we keep the old player image? also youre missing updates from main (camera tracking, background map)

File: app/project.godot (hunk 4 above)
Comment (abbclas) [reply app/project.godot]: do I just go to project.godot and delete?

Comment (noh0st) [reply app/project.godot]: yeah

**Review by noh0st — APPROVED** [review]
just be sure to push a commit to add the map back to the main scene before merging

**Pull request merged**